redesign search bar in top nav, expand footer (UI changes)

Krylan opened the pull request:
This commit introduces visual changes in top search bar, footer, but also updates theme styles and way of attaching them.

**Changes in Search UI**
![search-ui](https://user-images.githubusercontent.com/26091129/172256867-a8a1d4a2-6bcd-453e-a918-8c3a284b84bd.jpg)
I changed some styles of the standard browser inputs of top search bar, to fit it more into website design. Also, I decided to move theme selector to footer, to give more space in top bar (also, in footer there is a space for a suitable label for it).
Autocompletion box has unchanged styles. Changes in search bar concerns only the one in navigation bar – any other apperance of search on different pages has unchanged styles.

**Changes in footer**
![image](https://user-images.githubusercontent.com/26091129/172257268-627e2779-7282-4bc6-96a8-d288f6cfd9a2.png)
Footer background is now expanded to full width, while its content is still in width of main container. Theme selector is put above columns with links.

**Change the way of changing themes**
I noticed, that after selecting different theme, script was disabling already included themes and adding a new `<link>` element with theme CSS. It was creating unnecessary elements, so I changed it, assigning id "theme-style" to the `<link>` element of theme CSS file, and then changing only "src" attribute of it, in case of selecting a theme. I renamed the JS function "ResetTheme" to "changeTheme" to reflect the function contents. 
In case when `<link>` element with id "theme-style" doesn't exists (on the first visit), script will create it when changing theme for the first time.

**Change in theme styles**
I adapted theme styles to this changes, fixing by the way problem with some elements on "White" theme (tables had overriden background color from main CSS on even rows, and latest forum posts also had dark background).

TeddyWestside22 commented:
Wow, this is a massive improvement on the current search. Great job!

Maybe aligning the menu holder and search box with the below divs, as well as centering the menu items (to lessen the empty space) could make it even more visually appealing? It does already look awesome, but there's something satisfying about having things align. Here's an example:

![image](https://user-images.githubusercontent.com/32810074/172310962-0ac714e6-c96c-4497-9b4c-6cea45e63c89.png)

Only a suggestion though; I'd be very happy with either.

Jamiras reviewed (commented):
Seems to function, but I'll let someone with more css experience approve.

Krylan commented on public/css/styles.css:
```diff
@@ -892,6 +901,29 @@ td, tr, img {
+  color: #FFF;
+  height: 42px;
+  vertical-align: top;
+}
```
I expanded the input (adding more padding to it) to be easier to select by the user. That white outline on focus is default browser way to indicate, what element user has focused. It has strong color because of accessibility (it is especially helpful when you are navigating with keyboard and/or have visual impairement). If needed, it can be changed with "outline" CSS rule (it works similar to "border") on ":focus".
Magnifying glass as button is another focusable element, so I'm not sure if that's good idea to embrace it along (it would also require modifying the form, so the button would be over the text field).

Krylan commented on lib/constants.php:
```diff
@@ -1,4 +1,4 @@
 <?php
 
-const VERSION = '1.86.0';
+const VERSION = '1.88.0';
```
Oh, sorry for that, then.
Last times, when I was commiting some CSS/JS changes to the repo, there were some issues because of forgetting about version bump (so users were reporting problems because they had old versions of files in cache).
I was trying to remember about that, to not make this mistake again.
So if the way of updating the number changed since then, I will push the change to return to old number, along with other fixes.

Krylan commented:
Added a small update to the code: expanded search bar to fit right column in main container (as @TeddyWestside22 suggested), moved theme selector to the bottom-left of the footer (as @luchaos suggested) and returned the version number of resources in constant to its previous state (as @Jamiras pointed out).

I think it is ready to review and merge, unless there are any other comments about this changes.
